app/jobs/spree_springboard/import_variant_job.rb

```diff
@@ -5,9 +5,6 @@ class ImportVariantJob < ApplicationJob
     def perform
       import_manager = Spree::Variant.springboard_import_class.new
       import_manager.import_all_perform(SpreeSpringboard.client[:items])
-    rescue StandardError => error
```

Comment:
Why not add sentry in these 2 cases?

Reply:
It will be caught by Sentry in the stack.